### ISSUE TITLE
Issue with maphilight height and width properties when using IE

### ISSUE THREAD

**Issue by kbgould** [kemayo/maphilight#45]
Hope this doesn't come out a bit long winded here, but I ran into an error with this piece of code and have a potential fix.  Got here because I was using the timezone-picker thingy from timezonepicker.com

Basically I'm using angular with the ng-hide feature (and animations on it too).  When using IE if the element I'm looking to "maphilight" is not visible at the time that the code runs it isolates the height and width both as 0.  When we later make the items visible, this of course causes some issues.

Basically you call img.height and canvas.height etc. on the direct HTML DOM elements a lot and when they're not visibly rendered (at least in ie) they like to report themselves as having 0 height and width.  So instead I changed calls like:

var somethingArbitrary = img.height;

to:

var somethingArbitrary = $(img).height();

This got everything to work again.

Gonna try to fork it and submit my changes...
